source/getting-started/first-app.md

```diff
@@ -7,28 +6,29 @@ ember new my-new-app
 
 This will create a new `my-new-app` directory with your new Ember app inside.
 
-Once the generation process finishes, launch the Ember CLI development server:
+Once the process finishes, launch the Ember development server:
 
 ```shell
 cd my-new-app
 ember server
 ```
 
-Navigate to `http://localhost:4200` to see your new app in action.
+Navigate to [localhost:4200](http://localhost:4200) to see your app in action.
 
-The Ember CLI development server is great for developing your app, because it
+The Ember CLI development server
 provides features like rebuilding your app every time a file changes, and a
-mock server for setting up fake data for testing your app. In a production
-environment, though, features like these slow your app down; instead, you'll
-want to build a fast, optimized version of your app you can copy to your
-production server.
+mock server for setting up fake data when testing your app.
 
-To build your app for production, run:
+In a production environment, you'll
+want to build an optimized version of your website assets that can be copied to a
+server. To build your app for production, run:
 
 ```shell
 ember build --environment=production
 ```
 
-When this finishes running, the `dist/` directory will contain a version of your
-application suitable for use in production. You can then copy the contents of
-this folder to your production server.
+When this finishes the `dist/` directory will contain a version of your
+application suitable for production use. Copy the contents of
+this folder to your production server. If you are ready, explore the
+[ember-cli-deploy](http://ember-cli.github.io/ember-cli-deploy/) addon for
+more advanced functionality.
```

Comment:
@michaelrkn:
:+1: for "Alternatively, you can". But how about "For more advanced functionality, you can alternatively...". Maybe it will give the reader a leeway to decide to follow that lead.